header.adoc

```diff
@@ -0,0 +1,66 @@
+[[header]]
+:description: RISC-V ACPI Specification
+:revdate: 1/27/2022
+:revnumber: 0.2
+:revremark: This document is in Development state.  Change should be expected.
+:company: RISC-V.org
+:url-riscv: http://riscv.org
+:doctype: book
+:preface-title: Preamble
+:colophon:
+:appendix-caption: Appendix
+:title-logo-image: image:docs-resources/images/risc-v_logo.svg[pdfwidth=3.25in,align=center]
+// Settings:
+:experimental:
+:reproducible:
+// needs to be changed? bug discussion started
+//:WaveDromEditorApp: app/wavedrom-editor.app
+:bibtex-file: acpi.bib
+:bibtex-order: alphabetical
+:bibtex-style: apa
+:icons: font
+:lang: en
+:listing-caption: Listing
+:sectnums:
+:sectnumlevels: 5
+:toclevels: 5
+:toc: left
+:source-highlighter: pygments
+ifdef::backend-pdf[]
+:source-highlighter: coderay
+endif::[]
+:data-uri:
+:hide-uri-scheme:
+:stem: latexmath
+:footnote:
+:xrefstyle: short
+
+= RISC-V ACPI Platform Requirements Specification
+Sunil V L; RISC-V Platform HSC Group
+
+// Preable
```

Comment:
Oops, looks like we have a template problem: "Preable" -> "Preamble"

Reply:
Thanks will fix it.